.gitignore

```diff
@@ -137,3 +137,9 @@ dmypy.json
 
 # Poetry
 poetry.lock
+
+# Benchmark script
+*.csv
+*.txt
+*.md
+!README.md
```

Comment:
what dies the line !README.md mean?

Reply:
my benchmark script can generate markdown tables as visualization of the test results. to prevent that any of those are committed, I ignore all markdown-files *except* the `README.md`.